app/tasks/migrate.php

```diff
@@ -197,33 +36,36 @@ function fixDocument(Document $document) {
         $projects = [$console];
         $count = 0;
 
-        while ($sum >= 30) {
-            foreach($projects as $project) {
-                
-                $projectDB->setNamespace('app_'.$project->getId());
+        $migration = new Version\V06($register->get('db')); //TODO: remove hardcoded version and move to dynamic migration
 
+        while ($sum > 0) {
+            foreach ($projects as $project) {
                 try {
-                    $callbacks['0.5.0']($project, $projectDB);
+                    $migration
+                        ->setProject($project, $projectDB)
+                        ->execute();
                 } catch (\Throwable $th) {
                     throw $th;
-                    Console::error('Failed to update project ("'.$project->getId().'") version with error: '.$th->getMessage());
+                    Console::error('Failed to update project ("' . $project->getId() . '") version with error: ' . $th->getMessage());
```

Comment:
I like the spaces behind and after the `.`, but seems like the standard PHP formatter don't feel the same as bot of us. I should probably give it a run because its not automated (yet).

Reply:
Yeah, an old habit had me run code formatting in VS Code. Don't know what settings it used. Will revert 👍🏻